tests.js

```diff
@@ -11,8 +11,56 @@ chai.use(chaiChange)
 /*1: Guess The Number
 Write a program that will generate a number between 1 and 100. Your program will ask the user to guess the generated number.When the user types a guess, the program should tell them if they won (the guess matched the computer's number), if their guess was higher than the number the computer generated, OR if the guess was lower than the number the computer generated. Terminate the program if the user types "exit". */
 
```

Comment:
Is your `getComputerGuess()` function being imported and exported properly?

http://www.2ality.com/2014/09/es6-modules-final.html